packages/recipe-private-web-client/package.json

```diff
@@ -12,23 +12,23 @@
     "postinstall": "not-in-install && not-in-publish && npm run build-parent || true"
   },
   "dependencies": {
-    "@ciscospark/plugin-avatar": "^0.7.29",
-    "@ciscospark/plugin-board": "^0.7.29",
-    "@ciscospark/plugin-conversation": "^0.7.29",
-    "@ciscospark/plugin-encryption": "^0.7.29",
-    "@ciscospark/plugin-feature": "^0.7.29",
-    "@ciscospark/plugin-flag": "^0.7.29",
-    "@ciscospark/plugin-logger": "^0.7.29",
-    "@ciscospark/plugin-mercury": "^0.7.29",
-    "@ciscospark/plugin-metrics": "^0.7.29",
-    "@ciscospark/plugin-search": "^0.7.29",
-    "@ciscospark/plugin-support": "^0.7.29",
-    "@ciscospark/plugin-team": "^0.7.29",
-    "@ciscospark/plugin-user": "^0.7.29",
-    "@ciscospark/plugin-wdm": "^0.7.29",
-    "@ciscospark/spark-core": "^0.7.29",
-    "@ciscospark/storage-adapter-local-forage": "^0.7.29",
-    "@ciscospark/storage-adapter-local-storage": "^0.7.29",
+    "@ciscospark/plugin-avatar": "^0.7.30",
+    "@ciscospark/plugin-board": "^0.7.30",
+    "@ciscospark/plugin-conversation": "^0.7.30",
+    "@ciscospark/plugin-encryption": "^0.7.30",
+    "@ciscospark/plugin-feature": "^0.7.30",
+    "@ciscospark/plugin-flag": "^0.7.30",
+    "@ciscospark/plugin-logger": "^0.7.30",
+    "@ciscospark/plugin-mercury": "^0.7.30",
+    "@ciscospark/plugin-metrics": "^0.7.30",
+    "@ciscospark/plugin-search": "^0.7.30",
+    "@ciscospark/plugin-support": "^0.7.30",
+    "@ciscospark/plugin-team": "^0.7.30",
+    "@ciscospark/plugin-user": "^0.7.30",
+    "@ciscospark/plugin-wdm": "^0.7.30",
+    "@ciscospark/spark-core": "^0.7.30",
+    "@ciscospark/storage-adapter-local-storage": "^0.7.30",
+    "@ciscospark/storage-adapter-local-forage": "^0.7.30",
```

Comment:
Not sure if this matters, but the ordering got switched and they aren't alphabetical any longer.